conversion.go

```diff
@@ -99,6 +113,9 @@ func (z *Int) UnmarshalText(input []byte) error {
 // TODO: Ensure we have sufficient testing, esp for negative bigints.
 func (z *Int) SetFromBig(b *big.Int) bool {
 	z.Clear()
+	if b == nil {
+		return true
+	}
```

Comment:
In order to get a verdict, I checked what `big` does: 
```
func TestFoobar(t *testing.T) {
	new(big.Int).Set(nil)
}
```
==>
```
--- FAIL: TestFoobar (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
```

So, this is just defensive programming, let's remove it.

Reply:
cool. just removed.